corehq/apps/smsbillables/management/commands/bootstrap_unicel_outbound.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+import re
+import xlrd
+
+from django.core.management.base import LabelCommand
+
+from corehq.apps.accounting.models import Currency
+from corehq.apps.sms.models import OUTGOING
+from corehq.apps.smsbillables.models import SmsGatewayFee, SmsGatewayFeeCriteria
+from corehq.apps.unicel.api import UnicelBackend
+
+logger = logging.getLogger('accounting')
+
+FEE_COLUMN = 1
+COUNTRY_CODES_COLUMN = 3
+
+
+def bootstrap_unicel_outbound(orm):
+    currency = (orm['accounting.Currency'] if orm else Currency).objects.get(code="INR")
+    sms_gateway_fee_class = orm['smsbillables.SmsGatewayFee'] if orm else SmsGatewayFee
+    sms_gateway_fee_criteria_class = orm['smsbillables.SmsGatewayFeeCriteria'] if orm else SmsGatewayFeeCriteria
+
+    filename = (
+        os.getcwd()
+        + '/corehq/apps/smsbillables/management/pricing_data/Unicel-outgoing-codes.xls'
+    )
+    workbook = xlrd.open_workbook(filename)
+    table = workbook.sheet_by_index(0)
+
+    def get_codes(codes):
+        if type(codes) is float:
+            return [int(codes)]
+        codes = codes.replace(' ', '').replace(u'\xa0', '').replace('+', '')
```

Comment:
Perhaps irrelevant because you've looked at what the actual irregular characters are, but something like `codes = re.sub(r'[^\d,]', '', codes)` would be cleaner.